README.md

```diff
@@ -1,20 +1,10 @@
-![Repository Banner](https://raw.githubusercontent.com/elenajp/elenajp/main/me_diving.png)
+![](https://raw.githubusercontent.com/elenajp/elenajp/main/me_diving.png)
```

Comment:
I don't think this should be removed, it will cause a warning.
```suggestion
![Repository banner](https://raw.githubusercontent.com/elenajp/elenajp/main/me_diving.png)
```

Reply:
Empty alt means that nothing will be read by screen readers. That's what we generally want for decorative images/banners

Reply:
> Empty alt means that nothing will be read by screen readers. That's what we generally want for decorative images/banners

That creates a Markdown warning by the way. 🙂

Reply:
I do not think we are using markdown linter here + good functionality matters more than linter warning, also should be a comment to disable lint next line.